client/functions.lua

```diff
@@ -60,6 +60,33 @@ function QBCore.Functions.DrawText3D(x, y, z, text)
     ClearDrawOrigin()
 end
 
+function QBCore.Functions.CreateBlip(coords, sprite, display, scale, colour, shortRange, title)
+    print(coords, sprite, display, scale, colour, shortRange, title)
+    if not coords or not sprite or not display or not scale or not colour or shortRange == nil or not title then 
```

Comment:
What if somebody wants a blip without text? or without a colour etc?

Reply:
If a color pram isn't defined in a blip it'll just default to white, colour pram is required

Title will take an empty string fine, like I said in the PR this is something that I've been using without issues,

I'm taking all blip aspects into consideration so I have to check for everything, but feel free to PR something else as I only pushed this because I thought it'd be useful to others

Reply:
> If a color pram isn't defined in a blip it'll just default to white
So, it's not required then?